pvlib/iotools/epw.py

```diff
@@ -0,0 +1,178 @@
+"""
+Import functions for EPW data files.
+"""
+
+import io
+
+try:
+    # python 2 compatibility
+    from urllib2 import urlopen, Request
+except ImportError:
+    from urllib.request import urlopen, Request
+
+import pandas as pd
+
+
+def read_epw(filename=None, coerce_year=None):
```

Comment:
`filename` should be arg since we removed the `None` --> interactive load option.